_components/accordions.md

```diff
@@ -15,58 +15,84 @@ lead: An accordion is a list of headers that hide or reveal additional content w
-        </li>
-        <li>
-          <strong>The accordion uses javascript to set the <code>hidden</code> value of its content area.</strong> Each content area will have its <code>hidden</code> attribute set by the component, depending on its corresponding button&rsquo;s <code>aria-expanded</code> attribute. To ensure that your content is accessible in the event that the JavaScript does not load or is disabled, you should not manually set <code>hidden</code> on any of your content areas.
-        </li>
-      </ul>
-      <h4 class="usa-heading">Implementation</h4>
-      <ul class="usa-content-list">
-        <li>
-          <strong>Multiselectable accordion groups.</strong> Add the <code>aria-multiselectable="true"</code> attribute to any <code>usa-accordion</code> to create a multiselectable accordion group.
-        </li>
-        <li>
-          <strong>Default an accordion button to open.</strong> Add the <code>aria-expanded="true"</code> attribute to any <code>usa-accordion__button</code> to have that section open by default at page load.
-        </li>
-      </ul>
-      <h4 class="usa-heading">Package information</h4>
-      <ul class="usa-content-list">
-        <li>
-          <strong>Package usage:</strong> <code>@import usa-accordion</code>
-        </li>
-        <li>
-          <strong>Requires:</strong> <code>required</code>, <code>global</code>
-        </li>
-      </ul>
+          </li>
+          <li>
+            <strong>The accordion uses javascript to set the <code>hidden</code> value of its content area.</strong> Each content area will have its <code>hidden</code> attribute set by the component, depending on its corresponding button&rsquo;s <code>aria-expanded</code> attribute. To ensure that your content is accessible in the event that the JavaScript does not load or is disabled, you should not manually set <code>hidden</code> on any of your content areas.
+          </li>
+        </ul>
+        <h4 class="usa-heading">Implementation</h4>
+        <ul class="usa-content-list">
+          <li>
+            <strong>Multiselectable accordion groups.</strong> Add the <code>aria-multiselectable="true"</code> attribute to any <code>usa-accordion</code> to create a multiselectable accordion group.
+          </li>
+          <li>
+            <strong>Default an accordion button to open.</strong> Add the <code>aria-expanded="true"</code> attribute to any <code>usa-accordion__button</code> to have that section open by default at page load.
+          </li>
+        </ul>
+        <h5 id="component-settings">Accordion settings</h5>
+        {% assign settings = site.data.settings.components.accordion %}
+        {% include settings-table-simple.html
+          settings=settings.contents
+        %}
+        <h5 id="component-variants">Accordion variants</h5>
```

Comment:
Eventually we'll want to rewrite these tables and do this with data, but this is OK for now